### PR TITLE
Desing Update on Loading & LobbyList screen

### PR DESCRIPTION
**Pull request by Deudly**
ce49cf2cea4cb01851132cd46ef67006
Desing Update on Loading & LobbyList screen

### CODE REVIEW

**Comment by DeudlyYT** [comment]
@MythicManiac  Here you have a screenshot 
https://i.gyazo.com/ce49cf2cea4cb01851132cd46ef67006.jpg

**Comment by MythicManiac** [comment]
@DeudlyYT You can drag & drop screenshots here to github and they'll be embedded, just FYI

File: src/ui/css/lobby-list.less [hunk 1]
```diff
@@ -1,8 +1,45 @@
+@import "helpers";
+
+@login-background-image: url("http://screensaver.riotgames.com/latest/content/original/Champion%20Illustrations/Group%20Splashes/Blue_Buff_Fight.jpg");
+@login-background-blur: 3px;
+@login-background-darkness: 0.5;
+@font: "Lato";
+@header-height: 50px;
+
+.headerD{
```
Comment (MythicManiac) [inline src/ui/css/lobby-list.less:9]: Change this to something more descriptive

File: src/ui/css/lobby-list.less [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+        margin: 10px 20px 10px 20px;
+        padding-left: 10px;
+        padding-bottom: 5px;
+        
+        border-width: 0px;
+        border-bottom-width: 3px;
+        border-image: radial-gradient(ellipse at center, #BBBBBB 70%, rgba(0, 0, 0, 0));
+        border-image-slice: 1;
+}
+.titleD{
```
Comment (MythicManiac) [inline src/ui/css/lobby-list.less:23]: Change this to something more descriptive

File: src/ui/css/lobby-list.less [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+    color: #ffffff;
     
     & > div {
         cursor: pointer;
         -webkit-user-select: initial;
     }
-}
\ No newline at end of file
+}
+.circleA{
```
Comment (MythicManiac) [inline src/ui/css/lobby-list.less:40]: Change this to something more descriptive

File: src/ui/css/loading.less [hunk 4]
```diff
@@ -17,7 +20,9 @@
     .center-children;
 
     flex-direction: column;
-    background-color: rgb(84, 84, 185);
+    //background-color: rgb(84, 84, 185);
```
Comment (MythicManiac) [inline src/ui/css/loading.less:23]: Uncomment or remove

File: src/ui/components/lobbyList/lobbyListView.html [hunk 5]
```diff
@@ -1,9 +1,13 @@
-<input v-model="username" style="width: 100%; padding: 5px;">
-
 <div class="lobby-list">
-    <div v-for="lobby in network.lobbies" @click="join(lobby)">
+   <div class="headerD">
+        <span class="titleD">Choose a Game</span>
+   </div>
+   <div>
+        <input class="circleA" v-model="username" style="width: 100%; padding: 5px;">
```
Comment (MythicManiac) [inline src/ui/components/lobbyList/lobbyListView.html:6]: Change this classname to something more descriptive

File: src/ui/components/lobbyList/lobbyListView.html [hunk 6]
```diff
@@ -1,9 +1,13 @@
-<input v-model="username" style="width: 100%; padding: 5px;">
-
 <div class="lobby-list">
-    <div v-for="lobby in network.lobbies" @click="join(lobby)">
+   <div class="headerD">
+        <span class="titleD">Choose a Game</span>
```
Comment (MythicManiac) [inline src/ui/components/lobbyList/lobbyListView.html:3]: Change this classname to something more descriptive

File: src/ui/components/lobbyList/lobbyListView.html [hunk 7]
```diff
@@ -1,9 +1,13 @@
-<input v-model="username" style="width: 100%; padding: 5px;">
-
 <div class="lobby-list">
-    <div v-for="lobby in network.lobbies" @click="join(lobby)">
+   <div class="headerD">
```
Comment (MythicManiac) [inline src/ui/components/lobbyList/lobbyListView.html:2]: Change this classname to something more descriptive

**Comment by MythicManiac** [comment]
This definitely won't do for a final design, but I guess it's alright for a placeholder. You'll need to rename the classes to something more descriptive before I merge, though.

**Comment by Deudly** [comment]
@MythicManiac Changes you wanted are here https://github.com/LeagueSandbox/LobbyClient/pull/32

**Pull request closed without merging**